src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/algae/GregtechMTE_AlgaePondBase.java

```diff
@@ -82,6 +83,8 @@ protected GT_Multiblock_Tooltip_Builder createTooltip() {
 				.addInfo("Provide compost to boost production by one tier")
 				.addInfo("Does not require power or maintenance")
 				.addInfo("All Machine Casings must be the same tier, this dictates machine speed.")
+				.addInfo("All Buses/Hatches must, at least, match the tier of the Casings")
+				.addInfo("Place the controller after everything else to make sure it forms correctly!")
```

Comment:
I'm pretty sure if you add a pass with `minTierOfHatch = 100;` at the beginning of the structure definition/validation code, the placement order won't matter

Reply:
Oh, is that why the controller sometimes had to be placed last? I never figured it out.